hack/testing/assertions

```diff
@@ -14,3 +10,27 @@ assert_resources_exist(){
 	try_until_success "oc -n $NAMESPACE get elasticsearch elasticsearch" ${TIMEOUT_MIN}
 
 }
+
+assert_kibana_shared_config_exist() {
```

Comment:
didn't this move over to EO?

Reply:
This has been removed yes. This one probably remained after rebasing on the master. Anyway, it's not used anywhere. Removed.